.github/js/github-context-utils.js

```diff
@@ -1,19 +1,66 @@
-module.exports.getHeadCommitMessage = function(github) {
+module.exports.getCommitMessage = function(github) {
     const obj = JSON.parse(JSON.stringify(github));
-    let message = obj.event.commits[0].message;
-    if (obj.event.hasOwnProperty('head_commit')) {
-        const splitted = obj.event.head_commit.message.split("\n");
-        message = splitted[0];
-    }
-    return message;
-}
+    const commitMessage = (function() {
+        let message = '';
+        if (obj.event.hasOwnProperty('action')) {
+            message = getCommitMessageTriggeredByAction(github);
+        } else if (obj.event.hasOwnProperty('head_commit')) {
+            message = getHeadCommitMessage(github);
+        } else if (obj.event.hasOwnProperty('commits')) {
+            message = getOrdinaryCommitMessage(github);
+        } else {
+            // Do nothing
+        }
+
+        return message;
+        }());
+    return commitMessage;
+  }
 
-module.exports.getRepository = function(github) {
+  module.exports.getRepository = function(github) {
     const json = JSON.parse(JSON.stringify(github));
     return json.repository;
-}
+  }
 
-module.exports.getSHA = function(github) {
+  module.exports.getSHA = function(github) {
     const json = JSON.parse(JSON.stringify(github));
     return json.sha;
-}
+  }
+
+  function getCommitMessageTriggeredByAction(githubContext) {
+    const obj = JSON.parse(JSON.stringify(githubContext));
+    let commitMessage = '';
+    const action = obj.event.action;
+    commitMessage = "Erronous workflow was triggered by Action: " + action;
```

Comment:
Gir det mening å heller si at man ikke har funnet en commit meesage ved kjøring? Jeg godkjenner uansett, så kan du se hva du vil gjøre. Jeg er med på å bruke det du har fra før, det jeg foreslår, eller noe annet.

```suggestion
    commitMessage = "No commit message found in Action run: " + action;
```